src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -1,6 +1,11 @@
 declare -r TS_FMT='[%T%z] '
 declare -r REQS_NOT_FOUND_MSG='Could not find requirements.txt; Not running pip install'
 echo "Python Version: $python"
+PIP_CACHE_DIR=/usr/local/share/pip-cache
```

Comment:
Curious, why this particular location and not something like let's say `/tmp/pip-cache`?

Reply:
Can you kick off a Nightly build against this change? I am curious if this stuff works with `oryxtests/build` image? You probably already tried it?

Reply:
sure .. I know what you are trying to get .. but I added permission as we have for yarn cache, but yet. nightly-build is a good idea. Answering your first question, I tried to keep the location/path similar to the yarn-cache

Reply:
Cool